rom/dev/src/main.rs

```diff
@@ -115,7 +115,8 @@ extern "C" fn exception_handler(exception: &exception::ExceptionRecord) {
 
     // TODO: Signal non-fatal error to SOC
 
-    report_error(CaliptraError::ROM_GLOBAL_EXCEPTION.into());
+    let mut env = unsafe { rom_env::RomEnv::new_from_registers() };
```

Comment:
We shouldn't call this here, as some drivers (in particular, CSRNG), will need to do some initialization, which could cause some serious problems in a trap.

These zeroization calls are probably going to need to be unsafe static functions, similar to `Mailbox::abort_pending_soc_to_uc_transactions()`.